src/main/kotlin/com/pubnub/api/endpoints/FetchMessages.kt

```diff
@@ -103,8 +103,8 @@ class FetchMessages internal constructor(
             effectiveMax(page.limit, includeMessageActions, channels.size).toString()
         queryParams["include_uuid"] = includeUUID.toString()
 
-        page.start?.run { queryParams["start"] = this.toString().toLowerCase(Locale.US) }
-        page.end?.run { queryParams["end"] = this.toString().toLowerCase(Locale.US) }
+        page.start?.run { queryParams["start"] = this.toString().lowercase(Locale.US) }
```

Comment:
Why in some places we have:
Locale.getDefault() and here Locale.US ?

Reply:
That's a very good question and I don't know. It's not an issue for a start or an end, but I guess we should have the same locales in the code